08-Lucid-ORM/01-Getting-Started.adoc

```diff
@@ -716,6 +716,16 @@ await trx.rollback()
 
 ----
 
+[source, js]
+----
+const user = await User.find(1, trx)
+----
+
+[source, js]
+----
+const user = await User.query(trx).andWhere('username','virk).first()
```

Comment:
Shouldn't it be just `where` vs `andWhere`

Reply:
yeah